spec/requests/api/v1/journeys_controller_index_spec.rb

```diff
@@ -22,11 +22,12 @@
 
     let(:intermediate_journeys_count) { 1 }
     let(:page) { 1 }
-    let(:per_page) { 20 }
+    let(:per_page) { 5 }
     let(:url) { "/api/v1/moves/#{move.id}/journeys" }
+    let(:params) { {} }
 
     before do
-      get url, headers: headers, as: :json
+      get url, params: params, headers: headers
```

Comment:
when attempting to pass in `params`, the `as: :json` was changing the `GET` request to a `POST`